test/test_autograd.py

```diff
@@ -1320,25 +1348,46 @@ class dont_convert(tuple):
     (Atan, (), ((S, S, S),)),
     (Reciprocal, (), (torch.rand(S, S, S) + 0.1,)),
     (Cmax, (), ((S, S, S), (S, S, S))),
+    (Cmax, (), ((S, S, S), (S,)), 'broadcast_rhs'),
+    (Cmax, (), ((S,), (S, S, S)), 'broadcast_lhs'),
+    (Cmax, (), ((S, 1, S), (S, S)), 'broadcast_all'),
     (Cmin, (), ((S, S, S), (S, S, S))),
+    (Cmin, (), ((S, S, S), (S,)), 'broadcast_rhs'),
+    (Cmin, (), ((S,), (S, S, S)), 'broadcast_lhs'),
+    (Cmin, (), ((S, 1, S), (S, S)), 'broadcast_all'),
     (Round, (), ((S, S, S),)),
     (Sign, (), ((S, S, S),)),
     (Trunc, (), ((S, S, S),)),
     (Floor, (), ((S, S, S),)),
     (Ceil, (), ((S, S, S),)),
     (Frac, (), ((S, S, S),)),
     (Fmod, (), ((S, S, S), 1.5)),
+    (Fmod, (), ((S, S, S), Variable(torch.randn(S, S, S) + 2.5, requires_grad=False)), 'tensor'),
+    (Fmod, (), ((S, S, S), Variable(torch.randn(S) + 2.5, requires_grad=False)), 'tensor_broadcast_rhs'),
+    (Fmod, (), ((S,), Variable(torch.randn(S, S, S) + 2.5, requires_grad=False)), 'tensor_broadcast_lhs'),
+    (Fmod, (), ((S, 1, S), Variable(torch.randn(S, S) + 2.5, requires_grad=False)), 'tensor_broadcast_all'),
     (Lerp, (), ((S, S, S), (S, S, S), 0.2)),
+    (Lerp, (), ((S, S, S), (S,), 0.2), 'broadcast_rhs'),
+    (Lerp, (), ((S,), (S, S, S), 0.2), 'broadcast_lhs'),
+    (Lerp, (), ((S, 1, S), (S, S), 0.2), 'broadcast_all'),
     (Rsqrt, (), (torch.rand(S, S, S) + 1e-2,)),
     (Remainder, (), ((S, S, S), 1.5)),
+    (Remainder, (), ((S, S, S), Variable(torch.randn(S, S, S) + 2.5, requires_grad=False)), 'tensor'),
+    (Remainder, (), ((S, S, S), Variable(torch.randn(S) + 2.5, requires_grad=False)), 'tensor_broadcast_rhs'),
+    (Remainder, (), ((S,), Variable(torch.randn(S, S, S) + 2.5, requires_grad=False)), 'tensor_broadcast_lhs'),
+    (Remainder, (), ((S, 1, S), Variable(torch.randn(S, S) + 2.5, requires_grad=False)), 'tensor_broadcast_all'),
```

Comment:
It's better to use `rand` plus an eps to avoid division by zero. You can still easily get -2.5 from `randn`